### DISCUSSION TITLE
Documention Improvements

### DISCUSSION THREAD

**Discussion by nishantshah977** [Tungchaphet/messenger-api.js#7] [General]
Hey there, You must include everything we can do using this module in messenger like How to Send Attachments

Does parent Repository functions work while using it ?

**Comment by eptprofile** [comment]
The parent repository is deprecated, so some function in that is not working ;-;
Document still in progress ;-;

**Comment by 2tocom** [comment]
How to Send Attachments.
hello everyone, mình rất mong có một bản hướng dẫn cách gửi tệp đính kèm như hình ảnh, video, âm thanh hay tài liệu. Ai có cách sử dụng có thể đăng lên được không ạ

**Reply by eptprofile** [reply]
Như mình đã nói ở trên thì gửi tệp đính kèm đang có vẻ khá lỗi gì đó, tin tốt thì mình đã sửa được rồi nhưng mà chưa push code lên vì dạo này quá bận ;-;
Documents thì chắc phải chờ mình sửa xong E2EE của messenger thì mới làm tiếp đc ;-;

**Comment by nishantshah977** [comment]
![IMG_20231210_073342](https://github.com/misonomikadev/messenger-api.js/assets/99190466/fcac6812-8f66-4373-880a-5b8e5230cb11)

Meta is removing messenger from web

**Reply by eptprofile** [reply]
Wait what, this is mobile browser XD
